pkg/archive/paths.go

```diff
@@ -78,26 +80,33 @@ func WalkTarGzFiles(ctx context.Context, path string, walkFn FileWalkFn) (err er
 		}
 	}()
 
-	gzipReader, err := gzip.NewReader(file)
-
-	if err != nil {
-		return err
+	var reader *tar.Reader
```

Comment:
These changes can be removed in favour of https://github.com/palantir/log4j-sniffer/pull/34